generated/Client.php

```diff
@@ -1,11 +1,12 @@
 <?php
 
-declare(strict_types=1);
```

Comment:
Since your cs-fixer configuration does not enforce it, you will loose all `strict_types`, that's kind of sad here :/